src/functional/ffront/fbuiltins.py

```diff
@@ -64,6 +64,7 @@ def __gt_type__(self):
 # TODO(ricoh): This should be reunified with ``iterator.runtime.Offset``
 # potentially in ``functional.common``, which requires lifting of
 # ``ffront.common_types`` into ``functional``.
+# TODO(tehrengruber): remove inheritance since this is conceptually different to itir offset
```

Comment:
The two TODOs are contradicting each other.

If the latter applies, then it would require explanation why `Dimension` in field view is not also conceptually different from `Dimension` in iterator view.